src/modules/party/dto/acbs-post-party-request.dto.ts

```diff
@@ -0,0 +1,67 @@
+export interface AcbsPostPartyRequest {
+  PartyAlternateIdentifier: string;
+  IndustryClassification: {
+    IndustryClassificationCode: string;
+  };
+  PartyName1: string;
+  PartyName2: string;
+  PartyName3: string;
+  PartyShortName: string;
+  PartySortName: string;
+  PartyType: {
+    PartyTypeCode: string;
+  };
+  PrimaryAddress: {
+    AddressIdentifier: string;
+    AddressName1: string;
+    AddressType: {
+      AddressTypeCode: string;
+    };
+    Country: {
+      CountryCode: string;
+    };
+  };
+  DefaultGeneralLedgerUnit: {
+    GeneralLedgerUnitIdentifier: string;
+  };
+  CitizenshipClass: {
+    CitizenshipClassCode: string;
+  };
+  OfficerRiskDate: string | null;
```

Comment:
Can we use `DateString | null` please?

It doesn't actually change the underlying type restriction but at least it signifies intent